src/options.cpp

```diff
@@ -1479,6 +1479,16 @@ void options_manager::init()
 
     mOptionsSort["world_default"]++;
 
+    optionNames["any"] = _("Any");
+    optionNames["multiple"] = _("Multiple");
```

Comment:
"Multiple" sounds weird here, as if you could pick more than one.
"Multi-pool only" would be better.